rust/src/cli/ncl.rs

```diff
@@ -141,8 +141,9 @@ fn gen_conf(file_path: &str) -> Result<String, CliError> {
 
 #[derive(Clone, Debug, PartialEq, Serialize)]
 struct SortedNetworkState {
-    routes: Routes,
+    dns: DnsState,
     rules: RouteRules,
+    routes: Routes,
```

Comment:
Does the order matter for some reason?

Reply:
So that `nmstatectl-rust` is showing the same order as `nmstatectl-python`.